CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 97.1.1
 
 *  Add new args to get_content_by_embedded_document [PR](https://github.com/alphagov/gds-api-adapters/pull/1291)
```

Comment:
**suggestion:** detail

Might be useful to mention what these args are in the changelog?